inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/instrumentation/hook/HookManager.java

```diff
@@ -182,44 +160,49 @@ private void checkClassesForConfigurationUpdates(int batchSize) {
     private void updateHooksForClass(Class<?> clazz) {
         Map<MethodDescription, MethodHookConfiguration> hookConfigs = configResolver.getHookConfigurations(clazz);
 
-        val activeClassHooks = hooks.getUnchecked(clazz);
+        ConcurrentHashMap<String, MethodHook> activeHooks;
+        try {
+            activeHooks = hooks.get(clazz);
+        } catch (Exception e) {
+            //should actually never occur
+            log.error("Error updating method hook", e);
+            return;
+        }
 
-        deactivateRemovedHooks(clazz, hookConfigs, activeClassHooks);
-        addOrReplaceHooks(clazz, hookConfigs, activeClassHooks);
+        deactivateRemovedHooks(clazz, hookConfigs, activeHooks);
```

Comment:
seems you overwrote your own fixes in this class here, I am not even sure if this class should be changed at all in this commit?